.github/workflows/deploy-auto.yml

```
@@ -0,0 +1,25 @@
+# Automatically create a new version/tag, if new commits are available since the last tag
+name: Auto-tagger
+
+on:
+  push:
+    tags:
+      - deploy
+  schedule:
+    - cron: '51 15 */4 * *' # Every 4 days at 15:15 https://crontab.guru/#51_15_*/4_*_*
+
+jobs:
+  create-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

If we don't drop the `deploy` tag, we have to set this so it never gets stuck on "some random old commit that was tagged as `deploy` months ago"
```suggestion
      - uses: actions/checkout@v2
        with:
          ref: master
```